src/neuroconv/datainterfaces/behavior/audio/audiointerface.py

```diff
@@ -171,7 +171,7 @@ def add_to_nwbfile(
         write_as: Literal["stimulus", "acquisition"] = "stimulus",
         iterator_options: Optional[dict] = None,
         overwrite: bool = False,
-        verbose: bool = True,
```

Comment:
This verbose is actually unused (and redundant with the one in the init of the same interface), so it should be removed.